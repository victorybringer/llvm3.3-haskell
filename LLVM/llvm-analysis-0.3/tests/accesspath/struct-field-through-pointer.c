struct S {
  int x;
};

extern struct S *s;

void f() {
  s->x = 5;
}
