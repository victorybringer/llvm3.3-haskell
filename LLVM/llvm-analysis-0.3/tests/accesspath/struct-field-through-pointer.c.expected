fromList [ ("@f", ("%struct.S**", [AccessDeref, AccessDeref, AccessField 0])) ]
